Compiler/preproc/preprocessor.cpp

```diff
@@ -245,7 +245,9 @@ namespace Preprocessor {
                     size_t endOfString = FindIndexOfMatchingCharacter(text, i, text[i]);
                     if (endOfString == NOT_FOUND) //size_t is unsigned but it's alright
                     {
-                        LogError(ErrorCode::UnterminatedString, "Unterminated string");
+                        std::string msg = "Unterminated string: <char> is missing";
+                        msg = msg.replace(msg.find("<char>"), 6, std::string(1, text[i]));
```

Comment:
Formatting may be done using our String class, see String::Format or String::FromFormat.